src/extensions/scratch3_speech/index.js

```diff
@@ -691,23 +692,35 @@ class Scratch3SpeechBlocks {
             blocks: [
                 {
                     opcode: 'listenAndWait',
-                    text: 'Listen and Wait',
+                    text: formatMessage({
+                        id: 'speech.ListenAndWait',
+                        default: 'listen and wait',
+                        description: 'Start listening to the microphone and wait for data.'
+                    }),
                     blockType: BlockType.COMMAND
                 },
                 {
                     opcode: 'whenIHearHat',
-                    text: 'When I hear [PHRASE]',
+                    text: formatMessage({
+                        id: 'speech.whenIHear',
+                        default: 'when I hear [PHRASE]',
+                        description: 'Text on hat block that triggers when phrase is heard.'
+                    }),
                     blockType: BlockType.HAT,
                     arguments: {
                         PHRASE: {
                             type: ArgumentType.STRING,
-                            defaultValue: 'cat'
+                            defaultValue: 'cat' // TODO: Choose something.
                         }
                     }
                 },
                 {
                     opcode: 'getSpeech',
-                    text: 'speech',
+                    text: formatMessage({
+                        id: 'speech.speechReporter',
+                        default: 'speech',
+                        description: 'Text on speech reporter block.'
```

Comment:
These descriptions can be verbose, for use by translators, and this one-word block might be ambiguous (I guess the word "speech" might have many possible translations in different parts of speech etc). Let's try something like: 'Get the text of spoken words transcribed by the speech recognition system'

Reply:
done!